packages/dashmate/src/util/getOperatingSystemInfoFactory.js

```diff
@@ -0,0 +1,173 @@
+import si from 'systeminformation';
+import * as diskusage from 'diskusage';
+import os from 'os';
+
+export default function getOperatingSystemInfoFactory(
+  docker,
+) {
+  async function getOperatingSystemInfo() {
+    const result = {
+      cpu: null,
+      osInfo: null,
+      dockerSystemInfo: null,
+      currentLoad: null,
+      diskIO: null,
+      diskSpace: null,
+      fsOpenFiles: null,
+      inetLatency: null,
+      memory: null,
+    };
+
+    // Get System Info
+    try {
```

Comment:
Obfuscate home dir because it contains the username
   /home/ivanshumkov/ -> /home/******/)